src/azure-cli/azure/cli/command_modules/appservice/static_sites.py

```diff
@@ -211,14 +213,54 @@ def create_staticsites(cmd, resource_group_name, name, location,
         branch=branch,
         repository_token=token,
         build_properties=build,
-        sku=sku)
+        sku=sku_def)
 
     client = _get_staticsites_client_factory(cmd.cli_ctx)
     return sdk_no_wait(no_wait, client.create_or_update_static_site,
                        resource_group_name=resource_group_name, name=name,
                        static_site_envelope=staticsite_deployment_properties)
 
 
+def update_staticsite(cmd, name, resource_group_name=None, location=None,
+                      source=None, branch=None, token=None,
+                      app_location=None, api_location=None, app_artifact_location=None,
+                      tags=None, sku=None, no_wait=False):
+    existing_staticsite = show_staticsite(cmd, name, resource_group_name)
+    if not existing_staticsite:
+        raise CLIError("No static web app found with name {0}".format(name))
+
+    StaticSiteARMResource, StaticSiteBuildProperties, SkuDescription = cmd.get_models(
+        'StaticSiteARMResource', 'StaticSiteBuildProperties', 'SkuDescription')
+
+    if existing_staticsite.build_properties is not None:
+        app_location = app_location or existing_staticsite.build_properties.app_location
+        api_location = api_location or existing_staticsite.build_properties.api_location
+        app_artifact_location = app_artifact_location or existing_staticsite.build_properties.app_artifact_location
+
+    build = StaticSiteBuildProperties(
```

Comment:
we should avoid allowing these to be updated. Resource group and location also can't be updated we should avoid taking those in. It would be great to include: stagingEnvironmentPolicy, allowConfigFileUpdates

Reply:
removed properties which shouldn't be updated. synced offline that will add stagingEnvironmentPolicy, allowConfigFileUpdates in another PR after new SDK is released.